docker-compose/common-services/common-services.yaml

```diff
@@ -6,27 +6,84 @@ services:
     environment:
       - REFERENCE_SERVER_CONFIG_ENV=file:/config/reference-config.yaml
     depends_on:
-      - db
-      - kafka
+      - db_a
+      - kafka_a
     volumes:
       - .:/config
     ports:
       - "8081:8081" # reference-server
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+
+  anchor_platform_server_a:
+    image: stellar/anchor-platform:edge
+    build:
+      context: ../../
+      dockerfile: integration-tests/docker-compose-configs/Dockerfile
+    command: "--sep-server"
+    environment:
+      - STELLAR_ANCHOR_CONFIG=file:/config/anchor-platform-config.yaml
```

Comment:
This may confuse the examples. I don't think we should have `anchor_platform_server_a` as one of the common-services.

Reply:
Removed the anchor_platform_server_a